extensions.json

```
@@ -1148,14 +1150,12 @@
     {
       "id": "nwolverson.ide-purescript",
       "repository": "https://github.com/nwolverson/vscode-ide-purescript",
```

This change doesn't seem valuable. Please undo:

```suggestion
      "repository": "https://github.com/nwolverson/vscode-ide-purescript",
      "version": "0.23.3",
      "checkout": "v0.23.3",
```